src/jit/gentree.cpp

```diff
@@ -6308,6 +6308,10 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
```

Comment:
structDesc is a big structure and having a field of structDesc bloats size of call node excessively. How about GT_CALL node holding a ptr to SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR instead of the struct itself?

The added benefit is that it will simplify cloning of call nodes as well plus when we dynamically allocate default constructor will get invoked and structDesc will get initialized with invalid values.